namespace error when running helm lint

```
╰─±helm lint charts/rtm-api
==> Linting charts/rtm-api
[ERROR] templates/deployment.yaml: namespace option is currently NOT supported
[ERROR] templates/svc.yaml: namespace option is currently NOT supported
```

 Namespace is an acceptable stanza in the manifest yaml so why is it not supported in helm? If I have an app that is always installed into the same namespace I would rather not have to specify that in the CLI.

--- a/pkg/lint/rules/template.go
+++ b/pkg/lint/rules/template.go
@@ -104,8 +104,6 @@ func Templates(linter *support.Linter) {
 		if !validYaml {
 			continue
 		}
-
-		linter.RunLinterRule(support.ErrorSev, path, validateNoNamespace(yamlStruct))
 	}
 }
 
@@ -192,13 +190,6 @@ func validateNoMissingValues(templatesPath string, chartValues chartutil.Values,
 func validateYamlContent(err error) error {
 	if err != nil {
 		return fmt.Errorf("unable to parse YAML\n\t%s", err)
-	}
-	return nil
-}
-
-func validateNoNamespace(yamlStruct K8sYamlStruct) error {
-	if yamlStruct.Metadata.Namespace != "" {
-		return errors.New("namespace option is currently NOT supported")
 	}
 	return nil
 }

